mlos_bench/mlos_bench/storage/sql/experiment.py

```diff
@@ -89,7 +89,7 @@ def merge(self, experiment_ids: List[str]) -> None:
         _LOG.info("Merge: %s <- %s", self._experiment_id, experiment_ids)
         raise NotImplementedError()
 
-    def load(self, opt_target: Optional[str] = None) -> Tuple[List[dict], List[float], List[Status]]:
+    def load(self, opt_target: Optional[str] = None) -> Tuple[List[dict], List[Optional[float]], List[Status]]:
```

Comment:
Need any tests?

Reply:
we do have tests, and this is just a small issue in type annotation.. 
I wonder why it was not caught by mypy earlier, though